pontoon/base/models.py

```diff
@@ -267,7 +267,13 @@ def get_latest_activity(self, project=None):
             else:
                 return None
         else:
-            return ProjectLocale.get_latest_activity(project, self)
+            if hasattr(self, 'fetched_project_locales'):
```

Comment:
somehow i Feel we should document/marke this "remotely" initialized properties, because they're making this code magical. However, i'm not sure where's a good place for such docs. Any advices would be appreciated :)